packages/api/cli/src/electron-forge.ts

```diff
@@ -32,14 +32,12 @@ program.executeSubCommand = (argv: string[], args: string[], unknown: string[])
 program
   .version(metadata.version)
   .option('--verbose', 'Enables verbose mode')
-  .command('init', 'Initialize a new Electron application')
+  .command('init', 'Initialize a new Electron Forge application')
```

Comment:
question: is it necessary to specify that it's a Forge application here? It doesn't look like we do that for any other subcommand.